transport/transport.go

```diff
@@ -60,6 +67,11 @@ func NewHandler(as model.AuthService, us model.UserService, ps model.ProjectServ
 		})
 	})
 
+	h.Mux.Route("/organization/config", func(r chi.Router) {
+		r.Get("/", h.requireAuthUser(h.getConfig))
+		r.Patch("/", h.requireAuthUser(h.updateConfig))
```

Comment:
Oof.. I think `Settings` was good; `Config` might be confusing, because it is typically associated with "technical" server config, etc.
And for example Jira, a similar software has `Settings`.